.github/workflows/nodejs.yml

```diff
@@ -13,32 +13,31 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Use Node.js 14
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2-beta
```

Comment:
Why use the beta?

Reply:
https://github.com/actions/setup-node#v2-beta

It pulls Node from a cache. I have been using the beta in many repositories for months without issues.